src/js/other-websites/inpage_menu.js

```diff
@@ -356,100 +489,115 @@ const buildContent = {
       // API Note: If a user has not registered a subdomain yet, its default stored/queried value is "None";
       const isPremiumSubdomainSet = premiumSubdomainSet !== "None";
 
-      const masks = await getMasks({
+      // Get masks, including subdomain/custom masks if available
+      masks = await getMasks({
         fetchCustomMasks: isPremiumSubdomainSet,
       });
 
-      // Process the masks list:
-      if (masks.length === 0) {
-        fxRelayMenuBody.classList.remove("is-loading");
+      // Request the active tab from the background script and parse the `document.location.hostname`
+      const currentPageHostName = await browser.runtime.sendMessage({
+        method: "getCurrentPageHostname",
+      });
 
-        const search = document.querySelector(".fx-relay-menu-masks-search");
-        search.classList.add("is-hidden");
+      // See if any masks are assosiated with the current site. 
+      // If so, we'll end up building two discret mask lists:  
```

Comment:
🦅 👀 
```suggestion
      // If so, we'll end up building two discrete mask lists:  
```

Reply:
Revised in d5e6221fe717faa6809d549a9e0b85259f67a063